{{ cookiecutter.formal_name }}/briefcase.toml

```diff
@@ -3,6 +3,6 @@
 app_path = "x64/Release/app"
 app_packages_path = "x64/Release/app_packages"
 support_path = "x64/Release"
-{{ {"3.8": "support_revision = 10", "3.9": "support_revision = 13", "3.10": "support_revision = 7"}.get(cookiecutter.python_version|py_tag, "") }}
+{{ {"3.8": "support_revision = 10", "3.9": "support_revision = 13", "3.10": "support_revision = 8", "3.11": "support_revision = 0"}.get(cookiecutter.python_version|py_tag, "") }}
```

Comment:
See https://github.com/beeware/briefcase-windows-app-template/pull/3#pullrequestreview-1162391887.

Reply:
Same comment as on that PR; I've applied the dictionary reformatting.